apps/champions/README.md

```diff
@@ -24,3 +24,149 @@ end
 Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
 and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
 be found at <https://hexdocs.pm/curse_of_mirra>.
+
+## Useful Commands
+
+### Users
+
+#### Create User
+
+```
+{:ok, user} = Champions.Users.register("userName")
+```
+
+#### Get User by Id
+
+```
+user = Champions.Users.get_user(user_id)
+```
+
+#### Get User by Username
+
+```
+user = Champions.Users.get_user_by_username("userName")
+```
+
+#### Add Currency to User
+
+```
+{:ok, currency} = Champions.Currencies.add_currency(user.id, Currencies.get_currency_by_name!(currency_name).id, currency_amount)
+```
+
+---
+
+### Campaigns & Levels
+
+#### Get Campaigns
+
+```
+campaigns = Champions.Campaigns.get_campaigns()
+```
+
+#### Get Campaign
+Returns list of levels
+
+```
+campaign = Champions.Campaigns.get_campaign(campaign_number)
+```
+
+#### Get Level
+
+```
+level = Champions.Campaigns.get_level(level_id)
+```
+
+#### Fight Level
+
+```
+result = Champions.Battle.fight_level(user_id, level_id)
+```
+
+---
+
+### Units
+
+#### Create Character
+The parameters described are the required ones, the optional ones that can also be added are: rarity, base_health, base_attack, base_armor, basic_skill_id and ultimate_skill_id
+```
+{:ok, character} = GameBackend.Units.Characters.insert_character(%{game_id: Champions.Utils.game_id(), active: true, name: character_name, faction: faction_name})
+```
+
+#### Create Unit
+
+```
+GameBackend.Units.insert_unit(%{character_id: character_id, user_id: user_id, level: level, tier: tier, selected: false, slot: nil})
```

Comment:
I used this command and it returned an error, because the parameter is called `unit_level` instead of `level`

Did you test all the commands to make sure they worked?

Reply:
Actually nevermind, I think you may be testing it with an older version instead, since unit_level has already been renamed to level in this branch: https://github.com/lambdaclass/mirra_backend/blob/a2bd1d028857304f443072b5f7645f909cb64477/apps/game_backend/lib/game_backend/units/unit.ex#L16C1-L17C1

Reply:
Updated the PR with the latest main status and tested successfully. Resolving conversation!